SU2_CFD/src/output/CNEMOCompOutput.cpp

```diff
@@ -313,13 +317,16 @@ void CNEMOCompOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolv
 
   SetVolumeOutputValue("MOMENTUM-X", iPoint, Node_Flow->GetSolution(iPoint, nSpecies));
   SetVolumeOutputValue("MOMENTUM-Y", iPoint, Node_Flow->GetSolution(iPoint, nSpecies+1));
+  SetVolumeOutputValue("VELOCITY-X", iPoint, Node_Flow->GetSolution(iPoint, nSpecies+2));
```

Comment:
```suggestion
  SetVolumeOutputValue("VELOCITY-X", iPoint, Node_Flow->GetPrimitive(iPoint, nSpecies+2));
```
Similar as to the `CEulerVariable` class, for the `CNEMOEulerVariable` class, the velocity is stored in the `Primitive` vector, not the `Solution` vector.

Reply:
^^^
Additionally, the x-velocity will be at index "nSpecies", not "nSpecies+2"

Reply:
I have done this now